docs_src/design/adr/013-Device-Service-Events-Message-Bus.md

```diff
@@ -0,0 +1,251 @@
+
+```toml
+[MessageQueue]
+Protocol = 'tcp'
+Host = 'localhost'
+Port = 1883
+Type = 'mqtt'
+PublishTopicPrefix = 'edgex/events' # /<device-profile-name>/<device-name> will be added to this Publish Topic prefix
+SubscribeEnabled = true
+SubscribeTopic = 'edgex/events/#'
+[MessageQueue.Optional]
+    # Default MQTT Specific options that need to be here to enable evnironment variable overrides of them
+    # Client Identifiers
+    ClientId ="edgex-core-data"
+    # Connection information
+    Qos          =  "0" # Quality of Sevice values are 0 (At most once), 1 (At least once) or 2 (Exactly once)
+    KeepAlive    =  "10" # Seconds (must be 2 or greater)
+    Retained     = "false"
+    AutoReconnect  = "true"
+    ConnectTimeout = "5" # Seconds
+    SkipCertVerify = "false" # Only used if Cert/Key file or Cert/Key PEMblock are specified
+    Authmode = "none" # Valid values are: `none`, `usernamepassword`, `clientcert` or `cacert`
+    Secretpath = "messagebus"  # Path in secret store used if Authmode not `none`
+```
+
+#### Application Services
+
+##### [MessageBus]
+
+Similar to above, the Application Services `MessageBus` configuration will change to allow for secure connection to the MessageBus. The `Username` & `Password` have been replaced with `Authmode` & `SecretPath` for secure connections. See **[Secure Connections](#secure-connections)** section below for details.
+
+```toml
+[MessageBus.Optional]
+    # MQTT Specific options
+    # Client Identifiers
+    ClientId ="<app sevice key>"
+    # Connection information
+    Qos          =  "0" # Quality of Sevice values are 0 (At most once), 1 (At least once) or 2 (Exactly once)
+    KeepAlive    =  "10" # Seconds (must be 2 or greater)
+    Retained     = "false"
+    AutoReconnect  = "true"
+    ConnectTimeout = "5" # Seconds
+    SkipCertVerify = "false" # Only used if Cert/Key file or Cert/Key PEMblock are specified
+    Authmode = "none" # Valid values are: `none`, `usernamepassword`, `clientcert` or `cacert`
+    Secretpath = "messagebus"  # Path in secret store used if Authmode not `none`
+```
+
+##### [Binding]
+
+The `Binding` configuration section will require change for the subscribe topics scheme describe in the [MessageBus Topics](#messagebus-topics) section above to filter for Events from specific device profiles or devices. `SubscribeTopic` will change from a string property containing a single topic to the `SubscribeTopics` string property containing a comma separated list of topics. This allows for the flexibility for the property to be a single topic with the `#` wild card so the Application Service receives all Events as it does today.
```

Comment:
sp: section will require **changes** for the subscribe topics scheme **described** in the